samples/81.skills-skilldialog/dialog-root-bot/src/main/java/com/microsoft/bot/sample/dialogrootbot/Bots/RootBot.java

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MT License.
 
-package com.microsoft.bot.sample.dialogrootbot.bots;
+package com.microsoft.bot.sample.dialogrootbot.Bots;
```

Comment:
I'm not sure to add these changes in the PR, we are not fixing the DBI, we need to rollback this